Task 6/Task 6.2/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Task 6.2</title>
+</head>
+<body>
+    <form action="" class = "mainForm">
+        <p class = "formHeader">Регистрация</p>
+        <hr class = "registrationUnderline">
+        <div class = "e-mail"> 
+            <label for="e-mail">E-mail</label><br>
+            <input type="text" id = "e-mail">
+        </div>
+        <div class = "passwordContainter">
+            <div class = "password">
+                <label for="password">Пароль</label><br>
+                <input type="text" id = "password">
+            </div>
+            <div class = "passwordOnceAgain">
+                <label for="passwordOnceAgain">Ещё раз</label><br>
+                <input type="text" id = "passwordOnceAgain">
+            </div>
```

Comment:
You didn't make the inputs required - I can hit submit without filling these fields